### ISSUE TITLE
Don't need to show both extension's button and Legacy Tray icon

### ISSUE THREAD

**Issue by terrycloth** [codito/gnome-pomodoro#274]
The shell extension by default adds a button to the top panel, as well as an icon in the Legacy Tray. Thus, with an extension like TopIcons Plus, which moves the legacy tray icons to the shell panel, Pomodoro shows up in the panel twice.

![screenshot from 2017-02-23 16-39-23](https://cloud.githubusercontent.com/assets/3881885/23285588/0681a87e-f9e7-11e6-8576-593b05e19e07.png)

Even without TopIcons Plus, though, adding Pomodoro to both the top bar and to the Legacy Tray seems redundant.

My system setup:

* Fedora 25 Workstation x86_64
* GNOME Shell 3.22 in an X session
* gnome-shell-extension-pomodoro 0.13.0

**Comment by djfjeff** [comment]
I can confirm that with the latest version on Fedora 24 (0.13), gnome-pomodoro now appears on both the top bar and at the botton message tray, which is weird.

In previous version (0.12), it only appear in the top bar.

**Comment by terrycloth** [comment]
I also just noticed that I'm getting double notifications at the start of work time or break time... There's only one gnome-pomodoro process running, though.

```
USER      PID   PPID  NICE  %CPU  %MEM  STAT  ELAPSED  COMMAND
terrycl+  3063  1959  0     0.0   0.6   Sl    30:00    /usr/bin/gnome-pomodoro  --no-default-window
```

**Comment by kamilprusko** [comment]
Fixed in 0.13.1

It was reported in #256
